source/features/conflict-marker.tsx

```diff
@@ -45,7 +45,7 @@ async function addIcon(links: HTMLAnchorElement[]): Promise<void> {
 }
 
 function init(signal: AbortSignal): void {
-	observe('.js-issue-row:has(.octicon-git-pull-request.color-fg-open) a.js-navigation-open', batchedFunction(addIcon, {delay: 100}), {signal});
+	observe('.js-issue-row:not(:has(.octicon-git-pull-request-closed,.octicon-git-merge)) a.js-navigation-open[data-hovercard-type="pull_request"]', batchedFunction(addIcon, {delay: 100}), {signal});
```

Comment:
Can you export this to `selectors.ts` and add it to `selectors.test.ts`?

Reply:
Of course!
`data-hovercard` is need. If do not use it, issue types will be selected. But can select `pr-open` and `pr-draft` in this case.

Reply:
Then drop "not" and just include open and draft PRs. 

In this PR the change needed was to just add "octicon pr draft"